src/index.js

```diff
@@ -0,0 +1,156 @@
+// const axios = require("axios");
+
+// state which displays the temperature number and background of temperature number
+const state = {
+    tempNum: 75
+};
+
+// function to display temperature
+const displayTemp = () => {
+    const tempContainer = document.getElementById("tempValue");
+    tempContainer.textContent = `${state.tempNum}`;
+
+    tempContainer.className = ``;
+
+    if (state.tempNum > 79) {
+        tempContainer.className = `${tempContainer.className} red`;
```

Comment:
Suggestion: Since you clear the `className` on line 13, you don't get any benefit from prepending it here. This would have the same effect:

```suggestion
        tempContainer.className = 'red';
```